external/Kconfig

```diff
@@ -1,33 +1,34 @@
 
-config DM
-	bool "DM"
-	default n
-	depends on NET
-	---help---
-		enable the DM functionality
-if DM
 config LWM2M_WAKAAMA
```

Comment:
As such, it is not clear why we need to separate the build for wakaama and DM. DM framework is an essential part of wakaama. So, in case we build wakaama, the assumption goes that we build DM as well. In that case, the if DM statement is quite logical. Could you please explain why this is removed? Thanks.